ush/python/pygfs/ufswm/ufs.py

```diff
@@ -50,9 +60,84 @@ def parse_ufs_templates(input_template, output_file, ctx: Dict) -> None:
         matches = re.findall(pattern, file_out)
         if matches:
             logger.warn(f"{input_template} was rendered incompletely")
-            logger.warn(f"The following variables were not substituted")
-            print(matches)  # TODO: improve the formatting of this message
-        # TODO: Should we abort here? or continue to write output_file?
+            logger.warn(f"The following variables were not substituted:")
+            logger.warn(f"{', '.join(matches)}.")
 
         with open(output_file, 'w') as fho:
             fho.write(file_out)
+
+    @staticmethod
+    def __get_fixedfile_list(fixedfile_path) -> List:
```

Comment:
I don't know why we care about getting a list of all files in a directory.

Reply:
I thought this was what we discussed. In that path are only fixed-file YAMLs. They are relevant depending on the application of the GFS.